src/pages/scale.tsx

```diff
@@ -47,6 +47,10 @@ export function Scale({
   }
 
   let focusedHex: string | undefined;
+  const index =
+    Number(selectedIndex) > scale.colors.length - 1
+      ? String(scale.colors.length - 1)
+      : selectedIndex;
```

Comment:
This could be a bit more succinct with Math.min:

```ts
const index = Math.min(+selectedIndex, scale.colors.length - 1).toString()
```

Reply:
Fixed 😊